Runner as a service on Windows Environments

MegaDrive68k opened the pull request:
Here is the "Runner as a Service on Windows"  how to.

MegaDrive68k committed:
Automation Runner as a Service on Windows

MegaDrive68k committed:
Update runner-service-windows.md

Fixes.

MegaDrive68k committed:
Runner as a Service Images

fdevans commented on docs/learning/howto/runner-service-windows.md:
```diff
@@ -0,0 +1,210 @@
+
+To register the Automation Runner as a service, check the following steps:
+
+1. Download Apache Commons Daemons binaries from this location.
```
There is no link here.

fdevans commented on docs/learning/howto/runner-service-windows.md:
```diff
@@ -0,0 +1,210 @@
+
+## Resources
+
+[Automation Runner](https://docs.rundeck.com/docs/administration/runner/pre-4-11-runners.html#runner)
```
Local links should go to the `.md` file using the path after `/docs/`. in this case:

`/administration/runner/pre-4-11-runners.md#runner`

Using a direct link doesn't allow it to work within the staged or historical version sites properly.

MegaDrive68k replied:
Changing!

jsboak committed:
tuning content

jsboak committed:
add to sidebar menu

jsboak reviewed (approved):
made changes via commits. time to ship! 🚀

MegaDrive68k committed:
Update runner-service-windows.md

Includes:

1. Apache Daemons download link (line 111).
2. Right Runner doc entry in md format (Resources section).

MegaDrive68k committed:
Update runner-service-windows.md
